testdata/dnn/download_models.py

```diff
@@ -94,6 +94,16 @@ def save(self, r):
                 sys.stdout.flush()
 
 models = [
+    Model(
+        name='Alexnet Facial Keypoints (ONNX)', # https://github.com/ismalakazel/Facial-Keypoint-Detection
+        url='https://drive.google.com/uc?export=dowload&id=1etGXT9WQK1KjDkJ0pUTH-CaHHva4p9cY',
+        sha='e1b82b56b59ab96b50189e1b39487d91d4fa0eea',
+        filename='facial_keypoints.onnx'),
+    Model(
+        name='LightWeight Human Pose Estimation (ONNX)', # https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch
+        url='https://drive.google.com/uc?export=dowload&id=1T2Kq01WXzPMrQdnEOUEiVBhwouW8Pka5',
+        sha='20370164b8c43aa14625c0be4e31f62b9b397ac4',
+        filename='lightweight_pose_estimation.onnx'),
```

Comment:
Please move this to the end of this list.

Please put .onnx models in `onnx/models` subdirectory.